src/main/java/com/forgeessentials/economy/PlayerWallet.java

```diff
@@ -47,16 +47,22 @@ public boolean withdraw(long value)
     @Override
     public void add(long amount)
     {
-        this.amount += amount;
+        if (amount > 0)
```

Comment:
```java
if (value < 0) return false;
…
return true;
```

Reply:
After looking into it, to change add to return a boolean would require changing the wallet api interface.

It might be better to just remove the amount check since with this chance it would silently fail if amount was negative.  (It is important for withdraw that you have enough and you can not withdraw a negative number, but isn't as important for add)

@AnrDaemon do you think I should revert the code for add to `this.amount += amount;`

Reply:
The logic below has been reverted to the previous version since the base api for add is void and adding a negative check would require either throwing an exception or returning bool, and in both cases, would require changing an api that has been around since 1.7.10